netman/adapters/switches/remote.py

```diff
@@ -226,6 +226,18 @@ def set_bond_description(self, bond_number, description):
     def unset_bond_description(self, bond_number):
         self.delete("/bonds/" + str(bond_number) + '/description')
 
+    def set_interface_mtu(self, interface_id, mtu):
```

Comment:
The switch_base uses size instead of mtu. Should we stay consistent and keep it the same var name throughout?